c++/src/kj/exception.c++

```diff
@@ -1172,16 +1178,9 @@ struct FakeEhGlobals {
   uint uncaughtExceptions;
 };
 
-// LLVM's libstdc++ doesn't declare __cxa_get_globals in its cxxabi.h. GNU does. Because it is
-// extern "C", the compiler wills get upset if we re-declare it even in a different namespace.
-#if _LIBCPPABI_VERSION
-extern "C" void* __cxa_get_globals();
-#else
-using abi::__cxa_get_globals;
-#endif
 
 uint uncaughtExceptionCount() {
-  return reinterpret_cast<FakeEhGlobals*>(__cxa_get_globals())->uncaughtExceptions;
+  return std::uncaught_exceptions();
```

Comment:
Please use an `#if` here, otherwise this probably breaks some older environments.